plugins/projen/README.md

```diff
@@ -0,0 +1,31 @@
+# projen
+
+This plugin provides some aliases for [projen](https://github.com/projen/projen)
+
+To use it, make sure [projen](https://github.com/projen/projen) is installed, and add `projen` to the plugins array in your zshrc file.
+
+```zsh
+plugins=(... projen)
+```
+
+## Plugin commands
+
+* `pj_install` installs the `npm` module globally
+
+* `pgjn='projen new'`
+* `pgjv='projen --version'`
```

Comment:
Shouldn't these be `pjn` and `pjv`?

Reply:
pj = local via npx projen
pg = global (from npm -g)

so pgjv is the global projen version
and pjv is the local projen version

Reply:
That said, I'm happy to change any of them